src/Engine/ProtoCore/DSASM/DSasmDefs.cs

```diff
@@ -450,7 +450,7 @@ public struct Constants
         public const string kTempProcConstant = "temp_proc_var_";
         public const string kTempProcLeftVar = "%" + kTempProcConstant;
         public const string kImportData = "ImportData";
-        public const string kTempVarForNonAssignment = "temp_";
+        public const string kTempVarForNonAssignment = "temp6BBA4B28C5E54CF89F300D510499A00E_";
```

Comment:
Yah I got the idea after talking to you yesterday... if only we have a way to use an invalid identifier name just for internal consumption (i.e. only interpreter knows how to deal with these names internally).